include/h2o/http2_internal.h

```diff
@@ -164,6 +164,12 @@ struct st_h2o_http2_conn_t {
     /* internal */
     h2o_http2_scheduler_node_t scheduler;
     h2o_http2_conn_state_t state;
+    int chromium_dependency_tree; /* indicates whether the client-generated dependency tree is from Chromium
```

Comment:
Could we add "is_" prefix to this boolean attribute? I think we tend to do that.

Reply:
Sure. Will do.

Reply:
Done.